### DISCUSSION TITLE
http://localhost:4200/cours/4-full-cycle-dev-ops-%C3%A9levez-le-d%C3%A9veloppe-ment-au-del%C3%A0-du-build/21-lesson-2

### DISCUSSION THREAD

**Discussion by giscus** [zero-filtre/zerofiltre-blog#252] [Announcements]
# http://localhost:4200/cours/4-full-cycle-dev-ops-%C3%A9levez-le-d%C3%A9veloppe-ment-au-del%C3%A0-du-build/21-lesson-2

des petites bonnes actions.

http://localhost:4200/cours/4-full-cycle-dev-ops-%C3%A9levez-le-d%C3%A9veloppe-ment-au-del%C3%A0-du-build/21-lesson-2

**Comment by Shiriix** [comment]
test
